org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/graph/GraphPanelToolbar.java

```diff
@@ -503,10 +509,9 @@ private void updateSymbolButton(final String symbolOn, final String symbolOff,
     }
   }
 
-  private void assignTracks(final Layers layers)
+  private boolean assignTracks(final Layers layers)
```

Comment:
@saulhidalgoaular - could you include a javadoc to explain what the boolean status means?

Reply:
Good morning. Sure.